### PR TITLE
Sprint 2 Project

### PR DESCRIPTION
**Pull request by julianoh7**

### CODE REVIEW

File: src/test/resources/ResourcesForFeatureFiles/SprintTestCases.feature [hunk 1]
```diff
@@ -26,8 +26,8 @@ Feature: Verify
   Scenario: Verify error message for invalid sign in credentials
     Given I am on Hotels Homepage
     When  I click on Sign in
-    And   I enter invalid email
-    And   I enter invalid password
+    And   I enter email as 'jamesBond@gmail.com'
+    And   I enter password as 'Albania12345!'
     Then  I click signIn box
```
Comment (ahsansiddiki) [inline src/test/resources/ResourcesForFeatureFiles/SprintTestCases.feature:31]: Looks like this is an action step and should be labelled as **And**

File: src/test/resources/ResourcesForFeatureFiles/SprintTestCases.feature [hunk 2]
```diff
@@ -38,8 +38,8 @@ Feature: Verify
     When  click Search Box and type 'Bora'
     And   select 'Bora Bora'
     And   I click Check In
-    Then  I select Check-in date as '1'
-    Then  I select Check-out date from calendar as '15'
+    Then  I select Check-in date as '4'
+    Then  I select Check-out date from calendar as '20'
```
Comment (ahsansiddiki) [inline src/test/resources/ResourcesForFeatureFiles/SprintTestCases.feature:42]: Looks like above 2 lines are action steps and should be labelled as **And** and same observation for other Action steps.

File: src/test/resources/ResourcesForFeatureFiles/SprintTestCasesWeek2.feature [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    Given I am on Hotels Homepage
+    When  I click on Sign in
+    Then  I click on sign up
+    And   I enter email as 'james123'
+    And   I enter password as '123456789ABCDEFGH123456'
+    And   I enter name as '//.?'
+    And   I enter password as '123'
+    And   I enter last name as '.!@$%'
+    Then  I verify error email message is displayed
+    Then  I verify error password message is displayed
```
Comment (ahsansiddiki) [inline src/test/resources/ResourcesForFeatureFiles/SprintTestCasesWeek2.feature:14]: After First **Then** you can label the following verification steps as **And**

**Review by ahsansiddiki — APPROVED** [review]
Though I have some improvement comments but overall looks great !

**Comment by julianoh7** [comment]
Updated the work and will work towards improving my Gherkin.
Thank you

**Pull request merged**